forms/FieldList.php

```diff
@@ -149,8 +149,9 @@ public function removeFieldFromTab($tabName, $fieldName) {
 		$this->flushFieldsCache();
 
 		// Find the tab
-		$tab = $this->findOrMakeTab($tabName);
-		$tab->removeByName($fieldName);
+		$tab = $this->findTab($tabName);
+		if ($tab)
+			$tab->removeByName($fieldName);
```

Comment:
this should either be wrapped in curly braces or on one line (I'd prefer the former)